demos/demo-minimal-js/index.html

```diff
@@ -16,91 +17,155 @@
-        <div class="step">1. Initialize the calling app</div>
-        <input type="button" value="initialized" id="initialize" />
-        <div class="step">2. Log in to the app</div>
-        <input type="button" value="log in" id="login" disabled />
-        <input type="button" value="log out" id="logout" disabled />
-        <div class="step">3. Start a call</div>
-        <input type="button" value="outgoing call started" id="startcall" disabled />
-        <input type="button" value="call answered" id="answercall" disabled />
-        <div class="step">4. End a call</div>
-        <input type="button" value="call ended" id="endcall" disabled />
-        <input type="button" value="call completed" id="completecall" disabled />
+    <main>
+      <header>
+        <p class="instructions">
+          Click the buttons to send messages to HubSpot via the Calling
+          Extensions SDK. Open your console to see the incoming and outgoing
+          messages with HubSpot.
+        </p>
+      </header>
+      <section id="events">
+        <p>1. Initialize the calling app</p>
+        <input
+          id="initialize"
+          onclick="initialize()"
+          type="button"
+          value="initialized"
+        />
+        <p>2. Log in to the app</p>
+        <input
+          disabled
+          id="login"
+          onclick="logIn()"
+          type="button"
+          value="log in"
+        />
+        <input
+          disabled
+          id="logout"
+          onclick="logOut()"
+          type="button"
+          value="log out"
+        />
+        <p>3. Start a call</p>
+        <input
+          disabled
+          id="outgoingcall"
+          onclick="outgoingCall()"
+          type="button"
+          value="outgoing call started"
+        />
+        <input
+          disabled
+          id="answercall"
+          onclick="answerCall()"
+          type="button"
+          value="call answered"
+        />
+        <p>4. End a call</p>
+        <input
+          type="button"
+          value="call ended"
+          id="endcall"
+          disabled
+          onclick="endCall()"
+        />
+        <input
+          disabled
+          id="completecall"
+          onclick="completeCall()"
+          type="button"
+          value="call completed"
+        />
         <hr />
-        <div id="events">Other events</div>
-        <input type="button" value="send error" id="senderror" disabled />
-        <input type="button" value="change size" id="resize" disabled />
-      </span>
-    </div>
-    <script src="demo-minimal-js.bundle.js" async></script>
+        <p>Other events</p>
+        <input
+          disabled
+          id="senderror"
+          onclick="sendError()"
+          type="button"
+          value="send error"
+        />
+        <input
+          disabled
+          id="resize"
+          onclick="resize()"
+          type="button"
+          value="change size"
```

Comment:
Should we change the values to be consistent with the messages in the README? i.e. resize widget

Reply:
I've updated it to resize widget. The only other ones that don't match the SDK event names are `userLoggedIn`, `userLoggedOut`, and `outgoingCall`. Should I change those as well?

Reply:
Hmm the events are actually logged as `LOGGED_IN`, `LOGGED_OUT`, and `OUTGOING_CALL_STARTED` so we could probably update it here and in the README as well.

Reply:
Could you elaborate on what it would mean to update the SDK and demo apps to share the same values?

Reply:
Since the methods are `userLoggedIn` and `userLoggedOut` perhaps the events should be `USER_LOGGED_IN` and `USER_LOGGED_OUT`. Or we could deprecate the old method names and have `loggedIn` and `loggedOut` instead.

Reply:
I've created [CALL-3977](https://issues.hubspotcentral.com/browse/CALL-3977) to be done as part of Mainsail.